internal/authentication/tokengetter.go

```diff
@@ -19,6 +21,21 @@ type TokenGetter struct {
 	mu                 sync.RWMutex
 }
 
+type ServiceAccountNotFoundError struct {
+	ServiceAccountName      string // The name of the missing ServiceAccount.
+	ServiceAccountNamespace string // The namespace where the ServiceAccount should exist
+	Err                     error  // The underlying error
+}
```

Comment:
```suggestion
type ServiceAccountNotFoundError struct {
	ServiceAccountName      string
	ServiceAccountNamespace string
	Err                     error
}
```

Do we need the comments?

Reply:
removed the comments